src/status_im/ui/screens/accounts/recover/views.cljs

```diff
@@ -35,62 +59,66 @@
-       [status-bar/status-bar]
-       [toolbar/toolbar nil toolbar/default-nav-back
-        [toolbar/content-title (i18n/label :t/sign-in-to-another)]]
-       [react/view styles/inputs-container
-        [passphrase-input (or passphrase "") passphrase-error passphrase-warning]
-        [password-input (or password "") password-error (when-not disabled? sign-in)]
-        (when platform/desktop?
-          [react/i18n-text {:style styles/recover-release-warning
-                            :key   :recover-account-warning}])]
+       [cancel-link]
+       [step-i-of-n 1 2]
+       [recovery-phrase-title]
+       [recovery-phrase-explainer]
+       (if processing?
+         [passphrase-display passphrase]
+         [passphrase-input (or passphrase "") passphrase-error passphrase-warning])
+       (if (not (string/blank? (cofx :error))) [react/i18n-text {:style styles/recovery-error-message
+                                                                 :key   :recovery-error-message}])
+       (when platform/desktop?
+         [react/i18n-text {:style styles/recover-release-warning
+                           :key   :recover-account-warning}])
        [react/view components.styles/flex]
        (if processing?
          [react/view styles/processing-view
           [react/activity-indicator {:animating true}]
           [react/i18n-text {:style styles/sign-you-in
-                            :key   :sign-you-in}]]
+                            :key   :processing}]]
          [react/view {:style styles/bottom-button-container}
           [react/view {:style components.styles/flex}]
           [components.common/bottom-button
            {:forward?  true
-            :label     (i18n/label :t/sign-in)
+            :label     (i18n/label :t/next)
             :disabled? disabled?
             :on-press  sign-in}]])])))
+
+(defview recoverB [cofx]
+  [react/keyboard-avoiding-view {:style styles/screen-container}
+   [cancel-link]
+   [react/text {:style styles/key-recovered-title} (i18n/label :t/recovery-key-recovered-title)]
+   [react/text {:style styles/key-recovered-explainer} (i18n/label :t/recovery-key-recovered-explainer)]
+   [react/view styles/key-recovered-image-container
+    [react/view styles/key-recovered-image-circle
+     [photos/photo (cofx :photo-path) {:size 40}]]]
+   [react/text {:style styles/key-recovered-account-name} (cofx :name)]
+   [react/text {:style styles/key-recovered-account-address} (utils.core/truncate-str (cofx :address) 13 true)]
+   [react/view styles/key-recovered-reencrypt-your-key-button
+    [components.common/button {:button-style {:flex-direction :row}
+                               :on-press     #(re-frame/dispatch [:accounts.recover.ui/recover-account-enter-password (cofx :masked-passphrase) (cofx :accounts)])
+                               :label        (i18n/label :t/recovery-reencrypt-key)}]]])
+
+(defview recover [cofx]
+  (let [phase-num (cofx :phase)]
```

Comment:
Could you please replace integer values for phase with more descriptive keywords everywhere?

Reply:
Done!